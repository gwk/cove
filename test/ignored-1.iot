{
  'cmd': '$PROJ/cove.py test/ignored.py 1',
}
