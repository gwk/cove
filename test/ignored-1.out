cove report:

__main__: test/ignored-{}.py:
[30m   1 [37m  [0m
[30m   2 [0m  from sys import argv[0m
[30m   3 [37m  [0m
[30m   4 [37m  [0m
[30m   5 [0m  def top(on):[0m
[30m   6 [31m%   if on:[0m
[30m   7 [0m      inline(on)[0m
[30m   8 [0m      multi(on)[0m
[30m   9 [0m      partial(on)[0m
[30m  10 [37m  [0m
[30m  11 [36m| def inline(on): return on if on else None #no-cov![0m
[30m  12 [37m  [0m
[30m  13 [0m  def multi(on):[0m
[30m  14 [33m?   if on: #no-cov![0m
[30m  15 [33m?     return on[0m
[30m  16 [36m|   return None #no-cov![0m
[30m  17 [37m  [0m
[30m  18 [0m  def partial(on):[0m
[30m  19 [33m?   if on: #no-cov![0m
[30m  20 [33m?     return on[0m
[30m  21 [31m!   return None # never covered; this test assures that previous annotation is terminated.[0m
[30m  22 [37m  [0m
[30m  23 [37m  [0m
[30m  24 [0m  for a in argv[1]: top(int(a))[0m

__main__: test/ignored-{}.py: 24 lines; [37m8 trivial[0m; 16 traceable; 12 covered; 0 relaxed; [36m6 ignored[0m; [33m4 ignored but covered[0m; [31m2 not covered[0m.
